src/main/java/org/folio/circulation/domain/notice/session/PatronActionSessionService.java

```diff
@@ -118,12 +127,28 @@ public PatronActionSessionService(
     MultipleRecords<PatronSessionRecord> records) {
 
     if (records.isEmpty()) {
+      log.info("Records are empty, notices will not be sent.");
```

Comment:
Just by looking at this phrase in the log, it's not that easy to understand what exactly "Records" is. Should we maybe add so me more details?